requirements.txt

```diff
@@ -26,4 +26,6 @@ seaborn
 pykalman==0.9.5
 segmentation-models==1.0.1
 rich==10.16.1
-certifi>=2017.4.17,<=2021.10.8
\ No newline at end of file
+certifi>=2017.4.17,<=2021.10.8
```

Comment:
Do we need to version fence this on the upper end? I think some people were having issues with `certifi`